operations/bills.md

```diff
@@ -184,12 +245,27 @@ A bill is either a `Receipt` which means that it has been fully paid, or `Invoic
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `ItalianFiscalCode` | string  | optional | Italian fiscal code. |
+| `CustomerID` | string  | optional | ID of the Customer to whom the bill was assigned. |
+| `Address` | [Address](#address)  | optional | Address of the customer. |
+| `LegalIdentifiers` | Dictionary<string, string>  | optional | Legal Identifiers of the customer. |
+| `BillingCode` | string  | optional | ID of the Customer to whom the bill was assigned. |
+| `LastName` | string  | optional | Last name of the customer. |
+| `FirstName` | string  | optional | First name of the customer. |
+| `SecondLastName` | string  | optional | Second last name of the customer. |
+| `TitlePrefix` | string  | optional | Title prefix of the customer. |
+| `ItalianFiscalCode` | string  | deprecated | Italian fiscal code. Deprecated. Use values from LegalIdentifiers |
```

Comment:
```suggestion
| ~~`ItalianFiscalCode`~~ | ~~string~~ | ~~optional~~ | ~~Italian fiscal code.~~ Deprecated. Use values from `LegalIdentifiers` |
```

Reply:
Any deprecations such as this should be listed in the table on the [deprecations](../deprecations/README.md) page, so please update that page as well.

Reply:
@MikeAdamsMews I thought that if we deprecate it, we remove it from this table completely?

Reply:
@KaliCZ please confirm with @vitezp. We discussed changing the policy so that we use strikethrough for all deprecations. Regardless, they should be listed on the deprecations page.